### PR TITLE
german update

### PR DESCRIPTION
**Pull request by Henry2o1o**
:exclamation: Please review the [guidelines for contributing](https://github.com/skylot/jadx/blob/master/CONTRIBUTING.md#Pull-Request-Process)

### Description
Please describe your pull request.
Reference issue it fixes.

### CODE REVIEW

File: jadx-gui/src/main/resources/i18n/Messages_de_DE.properties [hunk 1]
```diff
@@ -40,9 +40,9 @@ tree.resources_title=Ressourcen
 tree.loading=Laden…
 
 progress.load=Laden
-#progress.export_mappings=
+progress.export_mappings=Zuordnungen exportieren
 progress.decompile=Dekompilieren
-#progress.canceling=Canceling
+progress.canceling=Abbrechen
```
Comment (jpstotz) [inline jadx-gui/src/main/resources/i18n/Messages_de_DE.properties:45]: This entry is shown after user has chosen to cancel a task. So in my opinion a better translation would be `breche ab`. It is used in this way: `progressPane.changeLabel(this, task.getTitle() + " (" + NLS.str("progress.canceling") + ")… ");`

File: jadx-gui/src/main/resources/i18n/Messages_de_DE.properties [hunk 2]
```diff
@@ -128,28 +128,28 @@ preferences.title=Einstellungen
...
-#preferences.useDebugInfo=Use debug info
+preferences.useDebugInfo=Debug-Infos verwenden
 preferences.inlineAnonymous=Anonyme Inline-Klassen
 preferences.inlineMethods=Inline-Methoden
 preferences.fsCaseSensitive=Dateisystem unterscheidet zwischen Groß/Kleinschreibung
 preferences.skipResourcesDecode=Keine Ressourcen dekodieren
-#preferences.useKotlinMethodsForVarNames=Use kotlin methods for variables rename
-#preferences.commentsLevel=Code comments level
+preferences.useKotlinMethodsForVarNames=Kotlin-Methoden für die Umbenennung von Variablen verwenden
+preferences.commentsLevel=Code-Kommentarebene
```
Comment (jpstotz) [inline jadx-gui/src/main/resources/i18n/Messages_de_DE.properties:152]: If I remember correctly this preferences entry manages how verbose the code comments are that are included into the decompiled Java code. I don't think someone will understand this by `Code-Kammentarebene`. What do you think of `Level der Code Kommentierungen`?

File: jadx-gui/src/main/resources/i18n/Messages_de_DE.properties [hunk 3]
```diff
@@ -166,7 +166,7 @@ preferences.start_jobs=Autom. Hintergrunddekompilierung starten
 preferences.select_font=Ändern
 preferences.select_smali_font=Ändern
 preferences.deobfuscation_on=Deobfuskierung aktivieren
-#preferences.deobfuscation_map_file_mode=Map file handle mode
+preferences.deobfuscation_map_file_mode=Datei-Handle-Modus zuordnen
```
Comment (jpstotz) [inline jadx-gui/src/main/resources/i18n/Messages_de_DE.properties:169]: This preference entry is about how Jadx handles the deobfuscation map files (values: `READ`, `READ_OR_SAVE`, `OVERWRITE`, `IGNORE`), so `Datei-Handle-Modus zuordnen` does not really fit here in my opinion. But I also don't have a good translation.

Comment (jpstotz) [reply jadx-gui/src/main/resources/i18n/Messages_de_DE.properties:169]: @Henry2o1o What do you think about `Map-Datei Modus`?

Comment (Henry2o1o) [reply jadx-gui/src/main/resources/i18n/Messages_de_DE.properties:169]: https://github.com/skylot/jadx/issues/1513

🤷‍♂️

Comment (jpstotz) [reply jadx-gui/src/main/resources/i18n/Messages_de_DE.properties:169]: @Henry2o1o Not sure what your comment means (sorry but I don't speak "smiley"). #1513 is about the removal of the preference `deobfuscationForceSave`.

File: jadx-gui/src/main/resources/i18n/Messages_de_DE.properties (hunk 1 above)
Comment (Henry2o1o) [reply jadx-gui/src/main/resources/i18n/Messages_de_DE.properties:45]: Breche ab, brechen sie ab, Abbrechen

File: jadx-gui/src/main/resources/i18n/Messages_de_DE.properties (hunk 3 above)
Comment (Henry2o1o) [reply jadx-gui/src/main/resources/i18n/Messages_de_DE.properties:169]: Umgang mit Map-Dateien
Handhabung der Map-Dateien

File: jadx-gui/src/main/resources/i18n/Messages_de_DE.properties (hunk 1 above)
Comment (jpstotz) [reply jadx-gui/src/main/resources/i18n/Messages_de_DE.properties:45]: @Henry2o1o In my opinion `Breche ab` would be best

**Comment by skylot** [comment]
@jpstotz, @Henry2o1o can I merge this PR?
It is hard for me to follow your discussion, so I just hope you will come to the conclusion :slightly_smiling_face:
Another option: I merge as is, and later someone will fix minor issues.

**Comment by jpstotz** [comment]
@Henry2o1o Could you please apply the changes we agreed on so that skylot can merge this PR?

**Comment by skylot** [comment]
So, no reply from @Henry2o1o :cry:

@jpstotz maybe you can add suggestions (like mentioned [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request#starting-a-review) in pt. 7), so I will be able to apply these changes and finally merge this PR :slightly_smiling_face:

**Review by jpstotz — APPROVED** [review]
@skylot An interesting way to modify the PR. I took the traditional way, by checking out the PR and then pushing it back to Henry2o1o:patch-8

**Comment by skylot** [comment]
@jpstotz great, thanks!

>  pushing it back to Henry2o1o:patch-8

I wasn't sure this available to collaborators, also PR contributor can forbid this, so sometimes it didn't work.
Anyway, thank you for help! :+1:

**Pull request merged**